drivers/hmis/app/models/hmis/hud/processors/base.rb

```diff
@@ -290,6 +290,28 @@ def normalize_single_custom_field_value(cded, value)
       else
         raise "unexpected value \"#{value}\""
       end
+    when 'file'
+      case value
+      when String
+        # If the value is a string, it should be a blob ID for a file that's just been uploaded to ActiveStorage,
+        # but doesn't yet have a File record in our database.
+        blob = ActiveStorage::Blob.find_by(id: value)
+        file = Hmis::File.new
+        file.name = blob.filename
+        file.client = @processor.send(:client_factory)
```

Comment:
This makes files added in an Assessment show up under Client Files, which may not be desirable.

Reply:
Yeah let's take another look at how we should structure this. We don't want them to show up there or in the warehouse.

Reply:
ok, I added a filter in the Client schema object, let me know if you think this is the right place to put it.